src/libs/ReportUtils.js

```diff
@@ -917,36 +917,34 @@ function buildOptimisticChatReport(
  */
 function buildOptimisticCreatedReportAction(ownerEmail) {
     return {
-        0: {
-            actionName: CONST.REPORT.ACTIONS.TYPE.CREATED,
-            reportActionID: NumberUtils.rand64(),
-            pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD,
-            actorAccountID: currentUserAccountID,
-            message: [
-                {
-                    type: CONST.REPORT.MESSAGE.TYPE.TEXT,
-                    style: 'strong',
-                    text: ownerEmail === currentUserEmail ? 'You' : ownerEmail,
-                },
-                {
-                    type: CONST.REPORT.MESSAGE.TYPE.TEXT,
-                    style: 'normal',
-                    text: ' created this report',
-                },
-            ],
-            person: [
-                {
-                    type: CONST.REPORT.MESSAGE.TYPE.TEXT,
-                    style: 'strong',
-                    text: lodashGet(allPersonalDetails, [currentUserEmail, 'displayName'], currentUserEmail),
-                },
-            ],
-            automatic: false,
-            sequenceNumber: 0,
-            avatar: lodashGet(allPersonalDetails, [currentUserEmail, 'avatar'], getDefaultAvatar(currentUserEmail)),
-            created: DateUtils.getDBTime(),
-            shouldShow: true,
-        },
+        reportActionID: NumberUtils.rand64(),
+        actionName: CONST.REPORT.ACTIONS.TYPE.CREATED,
+        pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD,
```

Comment:
A bug i noticed here and for buildOptimisticReportAction is that we are adding pendingAction here for both usages of this method doesn't have a success to clear the pendingAction. So even when I'm connected to the network the onyx data for this action shows up as pending. Its especially bad for `buildOptimisticReportAction` since whenever we turn off the network the recent chats show up as gray.

Is this something you can address for buildOptimisticCreatedReportAction and buildOptimisticReportAction?

Reply:
Done